Android/app/src/main/java/app/intra/sys/IntraVpnService.java

```diff
@@ -224,7 +224,12 @@ private void updateServerConnection() {
     if (equalUrls(url, null)) {
       // Use the Google Resolver
       AssetManager assets = this.getApplicationContext().getAssets();
-      newConnection = GoogleServerConnection.get(new GoogleServerDatabase(this, assets));
+      final GoogleServerConnection googleServerConnection =
```

Comment:
Just use newConnection here rather than creating a new variable?

Reply:
We need a `GoogleServerConnection` to call `didBootstrapWithFallback ` since `ServerConnection` does not define it.

Reply:
Ok, makes sense.